### PR TITLE
K8s links fix

### PR DESCRIPTION
**Pull request by evilnick**
## Done

 - Fixed some rendering issues in charm docs
 - updated links to Juju docs to avoid redirects
 - fixed relative links in charm docs
 - updated tigera links

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

### CODE REVIEW

**Comment by evilnick** [comment]
@anthonydillon 
I looked at https://github.com/canonical-web-and-design/ubuntu.com/pull/10202 and most of those fixes remained. A lot of new similar errors were introduced with creating the 1.22 docs recently though. The majority of these are due to mistakes in the actual charms, rather than the docs, but I will endeavour to fix them before we need to do 1.23

i also fixed a lot of Juju links to avoid having to go through 3 or 4 layers of redirection

File: templates/kubernetes/docs/1.20/charm-containerd.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Registry credentials. Setting this config allows Kubelet to pull images from
 registries where auth is required.
 
 The value for this config must be a JSON array of credential objects, like this:
-`[{"url": "https://my.registry:port", "username": "user", "password": "pass"}]`
+  ``[{"url": "https://my.registry:port", "username": "user", "password": "pass"}]``
```
Comment (anthonydillon) [inline templates/kubernetes/docs/1.20/charm-containerd.md:87]: Are you sure you want double quotes here?

**Review by anthonydillon — APPROVED** [review]
LGTM with one small comment

File: templates/kubernetes/docs/1.20/charm-containerd.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/1.20/charm-containerd.md]: Ah, well spotted. I was adding them to multiple places and I guess I did this one twice. fixed

**Pull request merged**